Bottom: a9b1b8987a4cd76c37363b128c9f1fffce531cfd
Top:    10dd504ac70bce8d2018afb785d59f8b89ddebc5
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 17:31:43 -0500

Fix credential copy to only run on daedalus

Credentials for all hosts end up on daedalus, so limit that play.


---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index eefa9706..81d9b004 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -129,7 +129,7 @@
     assemble_scripts: true
 
 - name: Cleanup tasks
-  hosts: all
+  hosts: daedalus
   remote_user: ansibleremote
   become: true
   become_user: root
@@ -137,14 +137,7 @@
   tasks:
   - name: Copy credentials to the /etc dir
     copy:
-      src: ./credentials/{{ inventory_hostname }}
-      dest: /etc/ansible-credentials/
-      owner: root
-      group: root
-      mode: 'u=rw,g=,o='
-  - name: Copy build system credentials to /etc dir
-    copy:
-      src: ./credentials/buildsystem.novatech-llc.com
+      src: "{{ playbook_dir }}/credentials/"
       dest: /etc/ansible-credentials/
       owner: root
       group: root
